src/monitoring/universal_pumpportal_listener.py

```
@@ -1,40 +1,64 @@
-            pump_program: Pump.fun program address
             pumpportal_url: PumpPortal WebSocket URL
+            platforms: List of platforms to monitor (if None, monitor all supported platforms)
         """
-        self.pump_program = pump_program
+        super().__init__()
         self.pumpportal_url = pumpportal_url
-        self.event_processor = PumpPortalEventProcessor(pump_program)
         self.ping_interval = 20  # seconds
+        
+        # Get platform-specific processors
+        from platforms.letsbonk.pumpportal_processor import LetsBonkPumpPortalProcessor
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)